### PR TITLE
Configure `.gitattributes` for the repository.

### PR DESCRIPTION
**Pull request by armiol**
@dmdashenkov PTAL.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+# Explicitly declare text files you want to always be normalized and converted
+# to native line endings on checkout.
+
+# Common formats
+*.html text
+*.xml text
+*.css text
+*.js text
+*.properties text
+*.bat text
```
Comment (dmdashenkov) [inline .gitattributes]: Why do we have `.bat` but don't have `.sh`?

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+# Action script and Flex
+*.as text
+*.mxml text
+*.actionScriptProperties text
+*.flexLibProperties text
+
+# Gradle build files
+*.gradle text
+gradlew.bat
```
Comment (dmdashenkov) [inline .gitattributes]: Same as above

File: .gitattributes [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+*.js text
+*.properties text
+*.bat text
+*.rtf text
+
+# SQL scripts
+*.sql text
+
+# Java sources
+*.java text
```
Comment (dmdashenkov) [inline .gitattributes]: Since we're in context of `tools`, let's also add `.groovy` here.

File: .gitattributes (hunk 1 above)
Comment (armiol) [inline .gitattributes]: Appended.

File: .gitattributes (hunk 3 above)
Comment (armiol) [inline .gitattributes]: Done.

**Comment by armiol** [comment]
@dmdashenkov PTAL.

**Pull request merged**